mcache/CHANGELOG.md

```diff
@@ -1,24 +1,19 @@
 # CHANGELOG - mcache
 
-1.2.0 / Unreleased
-==================
+## 1.3.0 / Unreleased
 
-### Changes
+* [FEATURE] Add support for SASL authentication.
```

Comment:
Is this the only one of these features thats unreleased. Odd that it was in the middle :)

Reply:
it was on purpose, I've changed the python client and we had issues ourselves so I reverted and merged again after 6.2

Reply:
Ahh perfect. Thanks for confirming.